### PR TITLE
Initial commit of azure supplemental values file

### PR DESCRIPTION
**Pull request by ts-reed-ni**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

This PR add the supplemental values file for azure app gateway ingress and service health paths

### Why should this Pull Request be merged?

Provide customers with the ingress and health check annotations template to speed up deployment of environments in Azure.

### What testing has been done?

Deployed an Azure environment manually using this supplemental values file and verified that the services start correctly with successful health checks.

### CODE REVIEW

File: getting-started/templates/Azure/azure_supplemental_values [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  ingress:
+    ## Ingress settings that apply to the apiHosts endpoints.
+    ##
+    api:
+      ## Annotations for the ingress.
+      ##
+      annotations: {
+        kubernetes.io/ingress.class: azure/application-gateway,
+        appgw.ingress.kubernetes.io/health-probe-path: "/up",
+        #appgw.ingress.kubernetes.io/appgw-ssl-profile: "stratus-dev",
```
Comment (BKnight760) [inline getting-started/templates/Azure/azure_supplemental_values:11]: Consider systemlink-ssl-profile instead

File: getting-started/templates/Azure/azure_supplemental_values [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        appgw.ingress.kubernetes.io/appgw-ssl-certificate: "stratus-dev-cert"
```
Comment (BKnight760) [inline getting-started/templates/Azure/azure_supplemental_values:12]: Consider systemlink-ssl-cert instead.

File: getting-started/templates/Azure/azure_supplemental_values [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      }
+    ## Ingress settings that apply to the hosts endpoints.
+    ##
+    ui:
+      ## @param global.ingress.ui.annotations Annotations for the ingress.
+      ##
+      annotations: {
+        kubernetes.io/ingress.class: azure/application-gateway,
+        appgw.ingress.kubernetes.io/health-probe-path: "/up",
+        #appgw.ingress.kubernetes.io/appgw-ssl-profile: "stratus-dev",
```
Comment (BKnight760) [inline getting-started/templates/Azure/azure_supplemental_values:22]: same name as above.

File: getting-started/templates/Azure/azure_supplemental_values [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        appgw.ingress.kubernetes.io/appgw-ssl-certificate: "stratus-dev-cert"
```
Comment (BKnight760) [inline getting-started/templates/Azure/azure_supplemental_values:23]: Same name as above.

**Review by JD-Robertson — APPROVED** [review]

File: getting-started/templates/Azure/azure_supplemental_values [hunk 5]
```diff
@@ -0,0 +1,67 @@
+global:
```
Comment (BKnight760) [inline getting-started/templates/Azure/azure_supplemental_values:1]: Add a comment block to the top that describes what this file is and how it should be used.

**Review by BKnight760 — CHANGES_REQUESTED** [review]

**Review by BKnight760 — APPROVED** [review]

**Pull request merged**